experimental/traffic-portal/src/app/api/cdn.service.ts

```diff
@@ -78,6 +78,28 @@ export class CDNService extends APIService {
 		return this.post<ResponseCDN>("cdns", cdn).toPromise();
 	}
 
+	/**
+	 * Queue updates to servers by a CDN
+	 *
+	 * @param id The CDN id to queue server updates for
+	 */
+	public async queueServerUpdates(id: number): Promise<CDNQueueResponse> {
+		const path = `cdns/${id}/queue_update`;
+		const action = {action: "queue"};
+		return this.post<CDNQueueResponse>(path, action).toPromise();
+	}
+
+	/**
+	 * Dequeue updates to servers by a CDN
+	 *
+	 * @param id The CDN id to dequeue server updates for
+	 */
+	public async dequeueServerUpdates(id: number): Promise<CDNQueueResponse> {
```

Comment:
these should also be able to take in `ResponseCDN` objects in addition to just the IDs thereof